parcels/particle.py

```diff
@@ -85,6 +85,9 @@ def __init__(self, lon, lat, grid, cptr=None):
         for var in self.user_vars:
             setattr(self, var, 0)
 
+        for var in self.user_vars:
+            setattr(self, var, 0)
+
     def __repr__(self):
```

Comment:
Why are these three new lines duplicated now? Or aren't they the same as the three lines above?